src/enochecker/checkerservice.py

```diff
@@ -142,6 +146,7 @@ class ExampleChecker(BaseChecker):
     flag_variants  = 1
     noise_variants = 1
     havoc_variants = 1
+    exploit_variants = 1
```

Comment:
Should we add an assert for `exploit_variants >= flag_variants`? :)

Reply:
don't think so, this just causes problems during development and is enforced by `enochecker_test`